spec/controler_spec.rb

```diff
@@ -2,15 +2,17 @@
 
 require 'controler'
 require 'null_action'
+require 'validator'
 
 RSpec.describe Controler do
   let(:actions) { [NullAction.new, NullAction.new] }
   let(:exit_choice) { UserInterface::EXIT_CHOICE }
   let(:output) { StringIO.new }
+  let(:validator) { Validator.new }
```

Comment:
you can just instantiate when needed, no need to create another variable in this case